### ISSUE TITLE
delite/handlebars.js should add modules required in templates to the build.

### ISSUE THREAD

**Issue by clmath** [ibm-js/delite#289]

**Comment by cjolif** [comment]
moving to the next milestone...

**Comment by wkeese** [comment]
Fixed by 2c0633fde5c8ef8e686b8a64209ae900ff356cba.

**Comment by wkeese** [comment]
I remember having to roll back the 2c0633f change to compile templates at build time, but handlebars.js still has code to inline the template requires into the build file:

```js
// Notify builder to take dependencies specified in requires="..." attribute, and inline them into the layer.
handlebars.addModules = function (pluginName, resource, addModules) {
	addModules(templateRequires);
};
```
